pythonforandroid/bootstraps/webview/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -63,7 +63,7 @@
 
         <activity android:name="org.kivy.android.PythonActivity"
                   android:label="@string/app_name"
-                  android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|fontScale|uiMode{% if args.min_sdk_version >= 8 %}|uiMode{% endif %}{% if args.min_sdk_version >= 13 %}|screenSize|smallestScreenSize{% endif %}{% if args.min_sdk_version >= 17 %}|layoutDirection{% endif %}{% if args.min_sdk_version >= 24 %}|density{% endif %}"
+                  android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|fontScale|uiMode|screenSize|smallestScreenSize|layoutDirection|density"
```

Comment:
The current conditionals are for the minimum SDK supported. In the `.p4a` configuration file we currently have:

```
--minsdk 21
```

So the only conditional that currently doesn't apply is `density`:

```
{% if args.min_sdk_version >= 24 %}|density{% endif %}
```

From docs https://developer.android.com/guide/topics/manifest/activity-element density was "Added in API level 24". So it seems right to keep the conditional in the template. We should instead bump our minimum SDK. SDK 21 is Android 5 which is way too old. I think we should bump to min SDK to 24.

Reply:
yeah, it's safe to bump to SDK 24, but then the condition is alwyas true, so potentialy could be removed as well

Reply:
No, because the upstream template should still work with older SDKs for other projects. Ideally we are going to get rid of this fork and use upstream p4a. The only reason we have p4a forked is because we have PRs upstream that are still open.